recipes/pycryptodomex/meta.yaml

```diff
@@ -0,0 +1,62 @@
+{% set name = "pycryptodomex" %}
+{% set version = "3.4.7" %}
+{% set sha256= "52aa2e540d06d63636e4b5356957c520611e28a88386bad4d18980e4b00e0b5a" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - toolchain
+  run:
+    - python
+
+test:
+  imports:
+    - Cryptodome
+    - Cryptodome.Cipher
+    - Cryptodome.Hash
+    - Cryptodome.IO
+    - Cryptodome.Math
+    - Cryptodome.Protocol
+    - Cryptodome.PublicKey
+    - Cryptodome.Random
+    - Cryptodome.SelfTest
+    - Cryptodome.SelfTest.Cipher
+    - Cryptodome.SelfTest.Hash
+    - Cryptodome.SelfTest.IO
+    - Cryptodome.SelfTest.Math
+    - Cryptodome.SelfTest.Protocol
+    - Cryptodome.SelfTest.PublicKey
+    - Cryptodome.SelfTest.Random
+    - Cryptodome.SelfTest.Signature
+    - Cryptodome.SelfTest.Util
+    - Cryptodome.Signature
+    - Cryptodome.Util
+
+about:
+  home: http://www.pycryptodome.org
+  license: Public Domain/BSD 2-Clause/OCB/MPIR
```

Comment:
Isn't MPIR some kind of LGPL? Not remembering version ATM. 😕 @isuruf?

Also what is OCB?

Reply:
Yeah, MPIR is LGPLv3+

Reply:
https://github.com/Legrandin/pycryptodome/blob/master/LICENSE.rst#ocb-license